hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/pipelines/pipelines.tsx

```diff
@@ -116,8 +116,8 @@ const COLUMNS = [
     render: (datanodes: string[]) => <div> {datanodes && datanodes.map(datanode =>
       <div key={datanode.hostName}>
         <div className='uuidtooltip'>
-          <Tooltip placement='top' title={`UUID: ${datanode && datanode.uuid}`} getPopupContainer={(triggerNode) => triggerNode}>
-            {datanode && datanode.hostName}
+          <Tooltip placement='top' title={`UUID: ${datanode?.uuid}`} getPopupContainer={(triggerNode) => triggerNode}>
+            {datanode?.hostName}
```

Comment:
It would be good to have a default fallback value.
```suggestion
            {datanode?.hostName ?? 'N/A'}
```

Reply:
Done changes in latest commit 878b46ef6f3090e43a104f606bdf49ad890f0e4f